docs/source/model_doc/tapas.mdx

```diff
@@ -0,0 +1,509 @@
+...         float_answer = batch["float_answer"]
+
+...         # zero the parameter gradients
+...         optimizer.zero_grad()
+
+...         # forward + backward + optimize
+...         outputs = model(input_ids=input_ids, attention_mask=attention_mask, token_type_ids=token_type_ids, 
+...                        labels=labels, numeric_values=numeric_values, numeric_values_scale=numeric_values_scale, 
+...                        float_answer=float_answer)
+...         loss = outputs.loss
+...         loss.backward()
+...         optimizer.step()
+```
+
+Equivalently, fine-tuning [`TFTapasForQuestionAnswering`] in native TensorFlow can be done as follows (shown here for the weak supervision for aggregation case):
+
+```
+>>> import tensorflow as tf
+>>> from transformers import TapasConfig, TFTapasForQuestionAnswering
+
+>>> # this is the default WTQ configuration
+>>> config = TapasConfig(
+...            num_aggregation_labels = 4,
+...            use_answer_as_supervision = True,
+...            answer_loss_cutoff = 0.664694,
+...            cell_selection_preference = 0.207951,
+...            huber_loss_delta = 0.121194,
+...            init_cell_selection_weights_to_zero = True,
+...            select_one_column = True,
+...            allow_empty_column_selection = False,
+...            temperature = 0.0352513,
+... )
+>>> model = TFTapasForQuestionAnswering.from_pretrained("google/tapas-base", config=config)
+
+>>> optimizer = tf.keras.optimizers.Adam(learning_rate=5e-5)
+
+>>> for epoch in range(2):  # loop over the dataset multiple times
+...    for batch in train_dataloader:
+...         # get the inputs; 
+...         input_ids = batch[0]
+...         attention_mask = batch[1]
+...         token_type_ids = batch[4]
+...         labels = batch[-1]
+...         numeric_values = batch[2]
+...         numeric_values_scale = batch[3]
+...         float_answer = batch[6]
+
+...         # forward + backward + optimize
+...         with tf.GradientTape() as tape:
+...              outputs = model(input_ids=input_ids, attention_mask=attention_mask, token_type_ids=token_type_ids, 
+...                        labels=labels, numeric_values=numeric_values, numeric_values_scale=numeric_values_scale, 
+...                        float_answer=float_answer )
+...         grads = tape.gradient(outputs.loss, model.trainable_weights)
+...         optimizer.apply_gradients(zip(grads, model.trainable_weights))
+```
+
+## Usage: inference
+
+Here we explain how you can use [`TapasForQuestionAnswering`] for inference (i.e. making predictions on new data). For inference, only `input_ids`, `attention_mask` and `token_type_ids` (which you can obtain using [`~transformers.TapasTokenizer`]) have to be provided to the model to obtain the logits. Next, you can use the handy [`~transformers.models.tapas.tokenization_tapas.convert_logits_to_predictions`] method to convert these into predicted coordinates and optional aggregation indices.
```

Comment:
```suggestion
Here we explain how you can use [`TapasForQuestionAnswering`] for inference (i.e. making predictions on new data). For inference, only `input_ids`, `attention_mask` and `token_type_ids` (which you can obtain using [`TapasTokenizer`]) have to be provided to the model to obtain the logits. Next, you can use the handy [`~models.tapas.tokenization_tapas.convert_logits_to_predictions`] method to convert these into predicted coordinates and optional aggregation indices.
```

First suggestion should be simplified like this, for the second we can remove transformers (and leave the ~ to hide the full path).